packages/benchmark-test/package.json

```diff
@@ -1,9 +1,27 @@
 {
-    "name": "benchmark",
-    "version": "1.0.0",
-    "type": "module",
-    "scripts": {
-        "test": "yarn rollup -c && eshost -h v8,xs dist/bundle.js"
-    },
-    "license": "MIT"
+  "name": "@agoric/benchmark-test",
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "build": "exit 0",
+    "prepack": "tsc --build tsconfig.build.json",
+    "postpack": "git clean -f '*.d.ts*' '*.tsbuildinfo'",
+    "test": "yarn rollup -c && eshost -h 'V8,Moddable XS' dist/bundle.js",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:eslint": "eslint .",
+    "lint:types": "tsc"
+  },
+  "author": "Agoric",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/Agoric/agoric-sdk/issues"
+  },
+  "homepage": "https://github.com/Agoric/agoric-sdk#readme",
```

Comment:
All cargo culted from patterns found in other packages. Note the license change from "MIT" to "Apache-2.0". It is our policy to use this license by default, unless there is a compelling reason for another choice. That said, I do like "MIT" ;)